environments/.env.mainnet

```diff
@@ -8,3 +8,4 @@ APP_WS_API_ETHERSCAN_TOKEN=FTKI6X11ZGTQ9J1KZ6WIRQ2ZW8EYZ8Y7FC
 APP_CURRENT_NETWORK=mainnet
 SENTRY_DSN=https://674c7230d4594037bc1a9797339aa831@o496053.ingest.sentry.io/5662609
 GTAG_ID=GTM-P9S4QSF
+APP_EXPLORER_VERSION=0.1.0
```

Comment:
I don't think we need to change this.

Reply:
I think we should keep this one, for the moment we deploy to mainnet